docs/nomad_simulations.md

```diff
@@ -1 +1,499 @@
+    ```python
+    scf.m_parent.jacobs_ladder
+    ```
+    which will return:
+    ```sh
+    'LDA'
+    ```
+
+## `ModelSystem` {#modelsystem}
+
+The `ModelSystem` section is an input section which contains all the information about the geometrical space quantities (positions, lattice vectors, volumes, etc) of the simulated system. This section contains various quantities and sub-sections which aim to describe the system in the most complete way and in a variety of cases, from unit cells of crystals and molecules up to microstructures. In order to handle this _hierarchical_ structure, `ModelSystem` is nested over itself, i.e., a `ModelSystem` can be composed of sub-systems, which at the same time could be composed of smaller sub-systems, and so on. This is done thanks to the (proxy) sub-section attribute called `model_system`.
+
+The `Cell` sub-section is an important section which contains information of the simulated cell. It does not necessarily contain the information of the elements sitting in each `positions`, as this is responsability of a more specialized section, the `AtomicCell`. This section contains information about the atoms conforming the material in the [`AtomsState`](#atomsstate) section. The `Symmetry` sub-section contains information about the symmetry of the system, and the `ChemicalFormula` sub-section about the strings that allow the system to be identified in a specific format of the chemical formulas (IUPAC, Hill, etc).
+
+The detailed relationship tree is:
+
+<div class="click-zoom">
+    <label>
+        <input type="checkbox">
+        <img src="../assets/part2-nomad-simulations/model_system.png" alt="ModelSystem quantities and functions UML diagram." width="80%" title="Click to zoom in">
+    </label>
+</div>
+
+The abstract section `Entity` is defined in the [Basic Formal Ontology (BFO)](https://basic-formal-ontology.org/) as a continuant entity, and we use it to abstract our `ModelSystem` using the intermediate base section `System`. This base section, `System`, is also used by the experimental data models.
+
+??? note "GeometricSpace and simulated cells."
+    The abstract section `GeometricSpace` is used to define more general real space quantities related with the system of reference used, areas, lengths, volumes, etc. However, this section and `Cell` are currently (July 2024) under revision and will probably change in the near future.
+
+
+### `AtomsState` and other sub-sections {#atomsstate}
+
+The `AtomsState` section is a sub-section of `AtomicCell` containing information about the specific chemical element used in the simulation which is located in each of the `positions` defined under `Cell`. It also contains information about the relevant `OrbitalsState`, as well as of `CoreHole` or `HubbardInteractions` information. The detailed relationship tree is:
```

Comment:
"...is a list of sub-sections within `AtomicCell`, corresponding to the list of particles specified by the `positions` array defined under `Cell`. Each `AtomicState` section contains information about a specific chemical element used in the simulation, and may also contain additional `OrbitalsState`, `CoreHole`, or `HubbardInteractions` information."